### PR TITLE
Banned functions

### PR DESCRIPTION
**Pull request by dristic**
This includes the rename fix for banned functions. The function is now called `bannedFunctions` everywhere.

@SimenB I could not find a branch called develop in the repository so this is against master. Maybe you have to push your develop branch?

Thanks!

### CODE REVIEW

File: src/checks/bannedFunctions.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+
+/**
+ * @description disallows use of a specific list of key words
+ * @param {string} [line] curr line being linted
+ * @return {boolean} true if translate3d is used, false if not
```
Comment (SimenB) [inline src/checks/bannedFunctions.js]: `translate3d`?

File: src/checks/bannedFunctions.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ */
+var bannedFunctions = function( line ) {
+	var found = false
+	var bannedFunctionList = this.config['bannedFunctions'] || []
```
Comment (SimenB) [inline src/checks/bannedFunctions.js]: `this.config.bannedFunctions`

**Comment by SimenB** [comment]
I dropped the develop branch approach, that's why you can't fin it 😄

**Comment by SimenB** [comment]
@dristic I can cherry pick it over into v2 myself if you don't want to, but at least make the 2 code changes 😄

File: src/checks/bannedFunctions.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	var index = -1
+
+	bannedFunctionList.forEach( function( func ) {
+		index = line.indexOf( func )
+		if ( index !== -1 ) {
+			found = true
+			this.msg( func + ' is currently banned', index )
```
Comment (SimenB) [inline src/checks/bannedFunctions.js]: `is currently banned` seems like a weird way to formulate it, IMO.

What about copying eslint's approach? So something like `unexpected function 'function-name' encountered`

**Comment by dristic** [comment]
Sounds good, let me make the changes and re-push.

**Comment by SimenB** [comment]
LGTM, thanks!

**Comment by SimenB** [comment]
@dristic Merged as https://github.com/SimenB/stylint/commit/4a6ff755279bbe119a9e8c94681ab7c7572b5382

**Comment by SimenB** [comment]
Your GH user didn't get credit for the commit since you haven't added you email to your account

image

**Comment by dristic** [comment]
Ah oh well. I'm using the wrong e-mail with this account. Maybe next time!

**Comment by SimenB** [comment]
@dristic My fault for forgetting, but could you update the README with the new option?

Make sure to branch from v2 😄

**Pull request closed without merging**